api/errors.js

```diff
@@ -48,6 +48,13 @@ let errors = {
     'title': 'IP address not authorized'
   },
 
+  'payment_required': {
+    'code': 402,
+    'detail': 'User does not have the required permission to perform that action',
```

Comment:
lacks permission? :P

Reply:
The detail field will always be overriden by the stripe object anyway, but I'll change it

Reply:
ah ok.